.github/workflows/pr_build.yaml

```diff
@@ -54,7 +56,7 @@ jobs:
       - uses: actions/checkout@v4
       - name: Update build number
         run: |
-          sed -i 's/Config.gameVersion = \".*\"/Config.gameVersion = \"${{ needs.gen-build-version.outputs.build }}\"/' "${env:GITHUB_WORKSPACE}/script/Config/App.lua"
+          sed -i 's/Config.gameVersion = \".*\"/Config.gameVersion = \"${{ needs.gen-build-version.outputs.build }}\"/' "${env:GITHUB_WORKSPACE}/script/Config/Version.lua"
```

Comment:
Should this change be committed?

Reply:
No, local development should always use a sentinel value like `0.0.0`. This version should only ever be seen in packaged builds downloaded by end users.